src/pages/LandingPages/MGCovid19/MGCovid19.vue

```diff
@@ -110,6 +114,20 @@ export default {
 	},
 	data() {
 		return {
+			co19HeaderTheme: {
+				themeKey: 'CO19',
```

Comment:
What does themeKey do?

Reply:
It's used for the cache in the footer. It' snot used for caching in the header as we don't cache that component.